@types/index.d.ts

```diff
@@ -176,28 +176,20 @@ declare class AbortError extends Error {
 	[Symbol.toStringTag]: 'AbortError';
 }
 
-
 declare function fetch(url: RequestInfo, init?: RequestInit): Promise<Response>;
-declare class fetch {
-	static default: typeof fetch;
-}
```

Comment:
I removed this class declaration, as it seems like it changes nothing when it comes to the types. Please let me know if I'm wrong here - maybe this was added intentionally due to a bug that is not covered by our types test suite.